### PR TITLE
streamline root, add stamp page

### PR DESCRIPTION
**Pull request by kjloveless**
add stamping to bsky

something quick to start sharing with others...
let me know what you think!

### CODE REVIEW

**Comment by kjloveless** [comment]
i noticed a bug in get "/login", but everything else seemed good.

File: apps/aqua/src/index.ts [hunk 1]
```diff
@@ -22,41 +24,60 @@ app.get("/client-metadata.json", (c) => {
...
-      return c.html(
-        `<div id="root">
-          <div id="header">
+  if (tealSession) {
+    const agent = await getSessionAgent(c);
+
+    const post = await agent?.getPost({
+      repo: "teal.fm",
+      rkey: "3lb2c74v73c2a",
+    });
```
Comment (mmattbtw) [inline apps/aqua/src/index.ts:36]: what is this :p (genuine question)

File: apps/aqua/src/index.ts [hunk 2]
```diff
@@ -22,41 +24,60 @@ app.get("/client-metadata.json", (c) => {
...
+    // const followers = await agent?.getFollowers();
+    return c.html(
+      `
+      <head>
+      <link rel="stylesheet" href="https://latex.vercel.app/style.css">
```
Comment (mmattbtw) [inline apps/aqua/src/index.ts:41]: can we serve this stylesheet ourselves orrrr

File: apps/aqua/src/index.ts [hunk 3]
```diff
@@ -122,6 +159,133 @@ app.post("/logout", (c) => {
...
+        <input
+          type="text"
+          name="track"
+          placeholder="track title (eg what's my age again?)"
+          required
+        />
+        <input
+          type="text"
+          name="link"
+          placeholder="https://www.youtube.com/watch?v=K7l5ZeVVoCA&pp=ygUdYmxpbmsgMTgyIHdoYXQncyBteSBhZ2UgYWdhaW4%3D"
```
Comment (mmattbtw) [inline apps/aqua/src/index.ts:205]:
```suggestion
          placeholder="https://www.youtube.com/watch?v=K7l5ZeVVoCA"
```

remove ugly query params :p

File: apps/aqua/src/index.ts [hunk 4]
```diff
@@ -122,6 +159,133 @@ app.post("/logout", (c) => {
...
+          <a href="/">home</a>
+          <a href="/stamp">stamp</a>
+        </div>
+        <form action="/logout" method="post" class="session-form">
+          <button type="submit" style="background-color: #cc0000; color: white; border: none; padding: 0rem 0.5rem; border-radius: 0.5rem;">logout</button>
+        </form>
+      </div>
+    </div>
+    <div class="container">
+      <p>want to share what you're listening to in the meantime??</p>
```
Comment (mmattbtw) [inline apps/aqua/src/index.ts:188]: imo, maybe we should explain that this is not the full product, like yeah "in the meantime" kinda gets that point across, but most will probably skip past that

something like:
> while we are building out our music tracker, try out this!

idk

**Review by mmattbtw — CHANGES_REQUESTED** [review]
it works! just have some nitpicks :3

**Review by espeon — COMMENTED** [review]
think matt said everything i wanted to point out - would mention that it's pretty easy to serve everything locally via hono

File: apps/aqua/src/index.ts (hunk 2 above)
Comment (espeon) [reply apps/aqua/src/index.ts:41]: something like this should work for serving the css and fonts for now https://hono.dev/docs/getting-started/nodejs#serve-static-files

**Review by mmattbtw — APPROVED** [review]

**Review by mmattbtw — DISMISSED** [review]

File: apps/aqua/src/auth/router.ts [hunk 5]
```diff
@@ -41,13 +42,82 @@ export async function callback(c: TealContext) {
...
   } catch (e) {
     console.error(e);
     return Response.json({ error: "Could not authorize user" });
   }
 }
 
+
+// Refresh an access token from a refresh token. Should be only used in SPAs.
+// Pass in 'key' and 'refresh_token' query params.
+export async function refresh(c: TealContext) {
```
Comment (espeon) [inline apps/aqua/src/auth/router.ts:63]: oops forgot to mount this on the router 🙃 should be fine for now though, we aren't using an SPA

**Review by espeon — APPROVED** [review]
lgtm, except for the thing i mentioned. hopefully later we can get the ui running on our frontend framework.

**Review by mmattbtw — APPROVED** [review]
lgtm

**Pull request merged**